fhir-persistence-schema/src/test/java/com/ibm/fhir/schema/derby/DerbyFhirDatabaseTest.java

```diff
@@ -162,4 +178,137 @@ protected void checkRefSequence(DerbyAdapter adapter) throws SQLException {
         assertNotNull(result);
         assertTrue(result >= FhirSchemaConstants.FHIR_REF_SEQUENCE_START);
     }
+    
+    /**
+     * Test table has data function.
+     *
+     * @param cp the cp
+     * @throws SQLException the SQL exception
+     */
+    protected void testTableHasDataFunction(IConnectionProvider cp) throws SQLException {
+        
+        
+        try (Connection connection = cp.getConnection()) {
+            try {
+                JdbcTarget tgt = new JdbcTarget(connection);
+                DerbyAdapter adapter = new DerbyAdapter(tgt);
+                
+                // validate table data when the table is empty
+                TableHasData cmd = new TableHasData("FHIRDATA", "evidence_logical_resources");
```

Comment:
You can use the SCHEMA_NAME final String value here.

Reply:
this has been fixed. Used SCHEMA_NAME final String value instead of "FHIRDATA"